torch/csrc/distributed/c10d/init.cpp

```diff
@@ -1807,6 +1807,14 @@ The hook must have the following signature:
               "group_name",
               &::c10d::ProcessGroup::getGroupName,
               "(Gets this process group name. It's cluster unique)")
+          .def_property_readonly(
+              "supports_splitting",
+              &::c10d::ProcessGroup::supportsSplitting,
+              "(test whether the backend supports splitting)")
```

Comment:
Ideally, `supports_splitting` should tie to Backend instead of ProcessGroup.

Reply:
I agree!  will change